### PR TITLE
Move table space management from migration scripts

### LINKED ISSUE #18: Only attempt to create tablespaces if they don't exist
Avoid creating tablespaces again, if they already exist, to prevent the mybatis migration scripts from failing (not good forcing the scripts to continue if one migration fails).

Since the changelog table is within the opensrp database, while tablespaces live outside this database, we end up in issues if the database is recreated (while the tablespaces still exist).

### PR DESCRIPTION
**Pull request by githengi**
Closes #18

### CODE REVIEW

File: assets/migrations/run_mybatis_migrations.sh [hunk 1]
```diff
@@ -1,13 +1,8 @@
...
-mkdir  /u01/postgres/feed
-mkdir  /u01/postgres/form
-chown -R postgres /u01/postgres
+
 unzip /tmp/mybatis-migrations-3.3.1-bundle.zip -d /opt/mybatis
 export MIGRATIONS_HOME=/opt/mybatis/mybatis-migrations-3.3.1
 MIGRATIONS=$MIGRATIONS_HOME/bin
 export PATH=$MIGRATIONS:$PATH
-migrate  up --path=configs/assets/migrations  --env=test --force 
+migrate  up --path=configs/assets/migrations  --env=test
```
Comment (jasonrogena) [inline assets/migrations/run_mybatis_migrations.sh]: @githengi how do you think we should test if the migrations are idempotent? Maybe run the `migrate up` command twice?

**Comment by jasonrogena** [comment]
Looks good to me. Thanks @githengi . This now means that it's up to the deployments to create the tablespace directories, if they'd want tablespaces, right?

File: assets/migrations/run_mybatis_migrations.sh (hunk 1 above)
Comment (githengi) [reply assets/migrations/run_mybatis_migrations.sh]: Okay, yes sure will add that

**Comment by githengi** [comment]
Yes so if deployments use tablespaces the tables spaces are created out of migrations just like creating database, users and roles.

**Comment by jasonrogena** [comment]
> Yes so if deployments use tablespaces the tables spaces are created out of migrations just like creating database, users and roles.

That's awesome, thanks!

**Comment by NtsikaMngoma** [comment]
I get an Error: Cause: org.postgresql.util.PSQLException: ERROR: tablespace "opensrp_core" does not exist when I run MyBatis migrate up flagged with the dir config path, test migration worked.  Only changelog has been created.

**Comment by githengi** [comment]
you have to create tablespaces externally of migrations or map the tablespaces to the postgres default tablespace

**Comment by NtsikaMngoma** [comment]
Okay, thanks a lot.

Kind Regards,

Sent from Outlook Mobile<https://aka.ms/blhgte>
________________________________

**Comment by NtsikaMngoma** [comment]
If possible please send me the link and admin test credentials that work for the Reveal App and for the OpenSRP web panel

Kind Regards,
Mark Mngoma

Sent from Outlook Mobile<https://aka.ms/blhgte>
________________________________

**Comment by NtsikaMngoma** [comment]
Reason  being, I have deployed the war but I get this blank screen shown in the image.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Pull request merged**